package.json

```diff
@@ -27,12 +27,16 @@
   "dependencies": {
   },
   "devDependencies": {
+    "boom": "^3.2.2",
     "chai": "^3.5.0",
     "eslint": "^1.10.3",
     "eslint-config-lob": "^2.2.0",
     "generate-changelog": "^1.0.2",
     "istanbul": "^0.4.4",
     "mocha": "^2.5.3",
     "hapi": "^13.4.0"
```

Comment:
i think you should also copy hapi as a peer dep since this plugin pretty much requires that hapi exists as well.

you should also list both peer deps (`then-redis` and `hapi`) as dev deps too so the test will work.